playkit/src/main/java/com/kaltura/playkit/Player.java

```diff
@@ -519,6 +520,13 @@
      */
     float getPlaybackRate();
 
+    /**
+     * get the Information for a thumbnailImage by position
+     *
+     * @param positionMS - relevant image for given player position.
+     */
+    ThumbnailInfo getThumbnailInfo(long positionMS);
```

Comment:
Let's change to `getThumbnail` to match existing web API.
The `positionMS` can be optional and if not given then return the current player position thumbnail info.

Reply:
@OrenMe  I do not see much different with this request.
this will make me use varargs internally

Reply:
@Override
    public ThumbnailInfo getThumbnailInfo(long ... positionMS) {
        log.v("getThumbnailInfo");
        if (assertPlayerIsNotNull("getThumbnailInfo()")) {
            if (positionMS.length != 0) {
                return player.getThumbnailInfo(positionMS[0]);
            } else {
                return player.getThumbnailInfo(player.getCurrentPosition());
            }
        }
        return null;
    }

Reply:
done